testdata/failing-worker.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+$fail = random_int(1, 100) < 10;
+$wait = random_int(1, 5);
+
+sleep($wait);
+if($fail) {
```

Comment:
```suggestion
if ($fail) {
```

Reply:
heh, can you tell that I didn't write this in phpstorm :)